contents/javascript/VueJs/Intro.md

```diff
@@ -0,0 +1,92 @@
+# VueJs
+
+Author: Lu Lechuan
+
+Vue.js is an open-source progressive [JavaScript framework](https://en.wikipedia.org/wiki/JavaScript_framework) for building user interfaces. It is designed to improve code quality and maintainablity.
+
+## Advantages of VueJs
+VueJs is an approachable, versatile and performant framework that helps to create a more maintainable and testable database.
+
+* Approachable: VueJs is very easy to learn. Comparing to other framework such as Augular, Vue is simple in term of API and design. Learning enough to build non-trivial applications typically takes less than a day.
```

Comment:
Did you mean trivial instead of non-trivial?

Reply:
Compared*

Reply:
I meant "non-trivial"